development/compiling/index.rst

```diff
@@ -1,3 +1,5 @@
+.. _doc_compiling:
```

Comment:
This should not be needed, you can reference `toc-devel-compiling` directly IINM.